Header header
uint8 target_idx
float32 posX
float32 posY
float32 velX
float32 velY
float32[] mds_pred_array
string prediction